Makefile

```
@@ -4,8 +4,11 @@ install:
 debug:
 	FLASK_APP=policyengine_api.api FLASK_DEBUG=1 flask run --without-threads
 
+test-env-vars:
+	pytest tests/unit/env_variables
+
 test:
-	MAX_HOUSEHOLDS=1000 coverage run -a --branch -m pytest tests --disable-pytest-warnings
+	MAX_HOUSEHOLDS=1000 coverage run -a --branch -m pytest tests/to_refactor tests/unit/services --disable-pytest-warnings
```

__minor, blocking:__ Please refactor a bit
```suggestion
	MAX_HOUSEHOLDS=1000 coverage run -a --branch -m pytest tests/to_refactor tests/unit --disable-pytest-warnings
```

I would make this change and move the environment variables tests to `tests/env_variables`, not as part of the `unit` folder. The `unit` folder will (hopefully) someday encompass many folders, and adding those manually will be more difficult than just adding `tests/unit`.